node/tessel-export.js

```diff
@@ -648,6 +655,13 @@ Tessel.I2C.prototype.transfer = function(txbuf, rxlen, callback) {
   this._port.uncork();
 };
 
+Tessel.I2C.prototype.disable = function() {
+  // Tell the coprocessor to disable this interface
+  this._port._simple_cmd([CMD.CMD_DISABLE_I2C]);
+  // Note that the bus is not enabled
+  this._port.I2C.enabled = false;
+};
+
```

Comment:
I don't have any specific cases - it seems like it might be nice for consistency with SPI and UART. I'll go ahead and remove so we can get this PR merged. If, in the slim chance, someone requests `I2C.disable`, we can add it then.